plugins/modules/host.py

```diff
@@ -225,6 +421,7 @@ def main():
             provision_method=dict(choices=['build', 'image', 'bootdisk']),
             image=dict(type='entity', scope=['compute_resource']),
             compute_attributes=dict(type='dict'),
+            interfaces_attributes=dict(type='list', elements='dict', options=interfaces_ansible_spec),
```

Comment:
`entity_list` does not work here, because this is not a list of entities, but a list of dicts that contain entities?
Do we need a new foreman_spec type? Why is `nested_list` not the right type?
Maybe we are "just" missing the lookup facilities inside a `nested_list`.

Reply:
Yeah, `nested_list` might work here.

Reply:
done